### ISSUE TITLE
Suggestion- compatibility with ICBM Classic

### ISSUE THREAD

**Issue by mzldh** [ValkyrienWarfare/Valkyrien-Warfare-Revamped#144]
A mobile missile launching platform has never been established before, sadly according to my testing ICBM Classic will not work on an airship, it always showed "target too far" for some reason. maybe it could be fixed by changing some code of positions and stuff.

here's a link to the ICBM page: https://minecraft.curseforge.com/projects/icbm-classic?gameCategorySlug=mc-mods&projectID=244451

**Comment by E-daw** [comment]
I highly suggest this to, this would be so incredibly cool

**Comment by mzldh** [comment]
apparently this have something to do with the locationing and coordinates of the blocks on ship, maybe this should be something to be mentioned.

**Comment by LemADEC** [comment]
ICBM classic only targets in the same world. A ship is essentially another world.

**Comment by LightningSh0ck** [comment]
@LemADEC wrong, a ship is a "projection" of blocks in the same dimension but far away

**Comment by LemADEC** [comment]
Sounds like the structure changed quite a bit since I've worked on it

**Comment by LemADEC** [comment]
WarpDrive is already 1.12

**Comment by Rubyjunk** [comment]
Hi, thanks for reporting this mod incompatibility issue! I appreciate every single report we get, so don't stop sending them. That said, we have a lot of work set out for us in release v1 of Valkyrien Warfare and therefore fixing game-breaking bugs and making the mod's experience more stable, fun, and consistent is our number one priority.

**What this means for you:** we might not be fixing this bug that soon. However, keep reporting these issues to us, it's really helpful and allows us to fix them when we get a chance.

This issue will be updated when work on it is started or it is obsoleted by a successive update. If you have any questions, feel free to ask them.

With <3,
Valkyrien Warfare Team
